include/baresip.h

```diff
@@ -643,7 +659,8 @@ typedef void (ua_exit_h)(void *arg);
 int  ua_alloc(struct ua **uap, const char *aor);
 int  ua_connect(struct ua *ua, struct call **callp,
 		const char *from_uri, const char *uri,
-		const char *params, enum vidmode vmode);
+		const char *params, enum vidmode vmode,
+		struct list *custom_hdrs);
```

Comment:
instead of breaking the `ua_connect` api, is it better to add a new function to set
the custom headers ? e.g. `ua_set_custom_hdrs()`